src/SearchField/index.jsx

```diff
@@ -185,7 +187,7 @@ class SearchField extends React.Component {
           onChange={this.handleChange}
           onKeyPress={this.handleKeyPress}
           inputGroupAppend={this.getSearchActionButtons()}
-          inputRef={(input) => { this.textInput = input; }}
+          inputRef={(input) => { if (!this.inputRef) { this.inputRef = input; } }}
```

Comment:
I think this should be pulled out of the render method and dealt with in the constructor:
`this.inputRef = this.props.inputRef || React.createRef()` so that this is just a straight `inputRef={this.inputRef}`

But we should wait for @adamstankiewicz review, as I'm not sure if there was a good reason this was being set in `render()`

Reply:
No, there wasn't a good reason for the `ref` being set in the `render` (personal preference). Totally cool if it's changed to what @georgebabey suggests. Probably makes more sense now since there's additional logic around whether to set the ref or not.

Reply:
@georgebabey Looks like Paragon is still on React 16.2 and createRef didn't come until 16.3. Not sure if I should updated Paragon to 16.3, or if there's a different way to do the `this.inputRef = this.props.inputRef || React.createRef()`